### ISSUE TITLE
⛔ 🚀 Feature Request: apply shadow without color specified

### ISSUE THREAD

**Issue by stijnvanouplines** [elementor/elementor#20486]
### Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

A shadow will not be applied to the element if there is no color specified.

### Describe the solution you'd like

But the [color](https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow#color) value of the `box-shadow` CSS property actually defaults to [currentColor](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#currentcolor_keyword).

```css
.button {
  box-shadow: 0 0 10px; /* no color specified */
  color: red; /* the shadow will be red */
}
```

### Additional context

So basically a shadow can be applied to an element without specifying a color. The shadow will get the color of the font by default.

**Comment by nicholaszein** [comment]
Hello @stijnvanouplines,

Thank you for submitting a feature request! 🙏 

### ⛔ After reviewing your request, we have decided not to add this to our roadmap for the time being.

It is not recommended to omit the color value on the box-shadow property. This might not be an official rule, but it's best practice to always declare all the values when using this property.

Changing this will also mean affecting backward compatibility, which is something we want to avoid.

🔖 We will, of course, keep this suggestion in mind and consider adding it if there is significant demand in the future.  

Please feel free to contact us again with other feature suggestions. 😉 

Best regards
